db/schema.rb

```diff
@@ -12,94 +12,95 @@
 
 ActiveRecord::Schema[7.0].define(version: 2022_01_13_124609) do
   # These are extensions that must be enabled in order to support this database
-  enable_extension "plpgsql"
+  enable_extension 'plpgsql'
```

Comment:
We do prefer double quotes for all now, updated rubocop. Please revert this back.